csrc/fusion_segmenter.h

```diff
@@ -339,10 +339,6 @@ class SegmentedFusion {
     return complete_fusion_->outputs();
   }
 
-  Val* findAlias(Val* val) const {
```

Comment:
It's only used in one place so I decided to inline it to avoid unnecessary indirection.